io.github.thetumultuousunicornofdarkness.cpu-x.yaml

```
@@ -0,0 +1,73 @@
+app-id: io.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: cpu-x
+rename-icon: com.github.thetumultuousunicornofdarkness.cpu-x.png
```

```suggestion
rename-icon: com.github.thetumultuousunicornofdarkness.cpu-x
```

is `rename-icon` not `rename-icon-file`